src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/flow_units/ResourceFlowUnit.java

```diff
@@ -128,16 +130,18 @@ public static ResourceFlowUnit buildFlowUnitFromWrapper(final FlowUnitMessage me
   public List<Field<?>> getSqlSchema() {
```

Comment:
Please add a javadoc comment for the public method while we are at it

Reply:
done